### ISSUE TITLE
Adding Dev Container Config to Workspace Folder when connected to WSL

### ISSUE THREAD

**Issue by patricknazar** [microsoft/vscode-remote-release#3039]
Issue Type: <b>Bug</b>

I'm using remote WSL in a workspace with two folders added. I tried to `Remote-Containers: Add Development Container Configuration Files...` and I got 

`Command 'Remote-Containers: Add Development Container Configuration Files...' resulted in an error (Command failed: "printenv" )`

I would have expected it to allow me to do this, no?

Extension version: 0.117.1
VS Code version: Code - Insiders 1.46.0-insider (2591ede95a52400d5d22989b16706f9b8b4831da, 2020-05-22T21:13:15.709Z)
OS version: Windows_NT x64 10.0.19041
Remote OS version: Linux x64 4.19.84-microsoft-standard

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 2700X Eight-Core Processor          (16 x 3693)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.92GB (0.86GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-18.04|
|OS|Linux x64 4.19.84-microsoft-standard|
|CPUs|AMD Ryzen 7 2700X Eight-Core Processor (4 x 3693)|
|Memory (System)|9.73GB (0.13GB free)|
|VM|0%|
</details>

**Comment by chrmarti** [comment]
What does the Dev Container log show? (`F1` > `Remote-Containers: Show Log`)

**Comment by patricknazar** [comment]
[5143 ms] Start: Run: wsl -d ubuntu-18.04 -e /bin/sh -c cd '/home/pat/git/ted-workspace.code-workspace' && printenv

**Comment by patricknazar** [comment]
[5143 ms] Start: Run: wsl -d ubuntu-18.04 -e /bin/sh -c cd '/home/pat/git/my-workspace.code-workspace' && printenv

**Comment by connor4312** [comment]
Tried to do this in WSL as well. Running the command did not create any output in the remote container "show log" or the devtools, but didn't seem to do anything. Using remote containers 0.120.0 and wsl 0.44.2 with today's Insiders.

**Comment by chrmarti** [comment]
@connor4312 Which WSL distro is this? What is the workspace folders structure? Where is the workspace file located? What is the output of `wsl -l -v`?

**Comment by patricknazar** [comment]
@chrmarti in my case:

> Which WSL distro is this?

WSL 2 using Ubuntu-18.04

> What is the workspace folders structure?

```
{
	"folders": [
		{
			"path": "folder1"
		},
		{
			"path": "folder2"
		},
		{
			"path": "..."
		}
	],
	"settings": {}
}
```

> Where is the workspace file located?

in the WSL file system, in my case `/home/pat/git/my-workspace.code-workspace`

> What is the output of `wsl -l -v`?

```
  NAME                   STATE           VERSION
* Ubuntu-18.04           Running         2
  docker-desktop-data    Running         2
  docker-desktop         Running         2
```

**Comment by connor4312** [comment]
I'm on Ubununtu 18.04 and WSL 1.x (where it looks like that command doesn't work)

**Comment by patricknazar** [comment]
I just tried it again, and it told me the file already exists:

`Dev Container configuration '/home/pat/git/.devcontainer/devcontainer.json' file already exists.`

This is a change, but I don't know how it happened (insiders update? Can't see any commits related). Presumably I had done it before somehow (maybe when not connected to WSL) in the same directory as the workspace so I deleted that folder and tried again and the action seemed to work. Any idea how this happened?

So I then tried:
- an individual open folder (no workspace); command worked;
- adding a folder to the workspace, thereby creating the workspace; command did not work anymore and no log produced;
- saving this new workspace and reopening it in a new window; command worked;
